openmoc/compatible/opencg_compatible.py

```diff
@@ -301,17 +302,34 @@ def get_compatible_opencg_surfaces(opencg_surface):
 
     surfaces = [left, right, bottom, top]
 
-  elif opencg_surface.type in ['x-cylinder', 'y-cylinder',
-                               'x-squareprism', 'y-squareprism']:
-    msg = 'Unable to create a compatible OpenMOC Surface from an OpenCG ' \
-          'Surface of type {0} since it is not compatible with OpenMOCs 2D ' \
-          'geometry formulation on the xy-plane'.format(opencg_surface.type)
-    raise ValueError(msg)
+  elif opencg_surface.type == 'y-squareprism':
+    x0 = opencg_surface.x0
+    z0 = opencg_surface.z0
+    R = opencg_surface.r
+
+    # Create a list of the four planes we need
+    left = opencg.XPlane(name=name, boundary=boundary, x0=x0-R)
+    right = opencg.XPlane(name=name, boundary=boundary, x0=x0+R)
+    bottom = opencg.ZPlane(name=name, boundary=boundary, z0=z0-R)
+    top = opencg.ZPlane(name=name, boundary=boundary, z0=z0+R)
+    surfaces = [left, right, bottom, top]
+
+  elif opencg_surface.type == 'z-squareprism':
```

Comment:
This should be `x-squareprism` with corresponding y and z surfaces. Also, how do you feel about changing left, right, bottom, and top to ymin, ymax, zmin, and zmax and making this consistent throughout this compatibility module? With the move from strictly 2D in xy to 3D, the bottom and top references can be ambiguous since they could refer to y or z.